### DISCUSSION TITLE
Possible to use FlyntWP without Timber?

### DISCUSSION THREAD

**Discussion by josh-tt** [flyntwp/flynt#394] [Q&A] [answered]
Managing a personal project now (not a dev) and I am wondering if Timber is a requirement for using Flynt if we're using custom components. Currently we have lots of templates written for ACF with php and don't have much time to deal with an added layer.  Thanks

**Comment by steffenbew** [comment, marked as answer]
Hi @josh-tt, we're using Timber a lot in our Flynt Starter Theme. Unbundling that connection is certainly possible, but would involve quite some custom work.

**Reply by josh-tt** [reply]
Thanks Steffen. Much appreciated. Looks great, just in a bit of a rush and not sure if adding something new to the mix will help or not this time around. Cheers
